<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans SemiBold</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: Open Sans SemiBold;
	font-color: white;
}


QPushButton{
    background-color: #2e9cca;
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color: #aaabb8;
	color: white;
}

QPushButton:pressed {
	border-width: 5px;
	border-color: white;
	
}

QPushButton:hover{
	background-color: #2964BA;
}

QPushButton:disabled{
	background-color: #464866
}

#centralwidget{
	background: #2A1B3D;
}

QLineEdit{
	background-color: #aaabb8;
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color: white;
	selection-background-color: dark grey;
	color: black;
	padding: 2px
}

QLabel{
	color: white;
	font: bold;

}

QTableWidget{
	background-color: #464866;
	color: white;
	selection-background-color: #2964BA;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 1px;
	border-radius: 15px;
	gridline-color: #aaabb8;
}

QRadioButton{
	color: white;
}

QComboBox{
	background: #464866;
	color: white;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 3px;
	border-radius: 10px;
	font-size: 16px;
}

QComboBox:hover{
	background: #2964BA
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="q_title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>34</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_username">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Username or Email</string>
    </property>
   </widget>
   <widget class="QWidget" name="q_signin" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>221</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QPushButton" name="q_signin_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans SemiBold</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
    <widget class="QLabel" name="q_password_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans SemiBold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="q_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="q_dont_have_account">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>I don't have an account</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="q_create_account" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QPushButton" name="q_create_account_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans SemiBold</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="q_make_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q_repeat_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="q_repeat_password_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans SemiBold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Repeat Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="q_make_password_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans SemiBold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="q_show_password">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
